tools/ci/tc/github_checks_output.py

```diff
@@ -17,26 +16,21 @@ class GitHubChecksOutputter(object):
     https://docs.taskcluster.net/docs/reference/integrations/github/checks#custom-text-output-in-checks
     """
     def __init__(self, path):
-        # type: (Text) -> None
+        # type: (str) -> None
         self.path = path
 
     def output(self, line):
-        # type: (AnyStr) -> None
-        text = ensure_text(line)
-        # NOTE: mypy types the "text mode" of open() in Python 2 as BinaryIO,
-        # which makes sense as we cannot specify its encoding (it's
-        # platform-dependent), while io.open() is closer to open() in Python 3.
-        # TODO: use the built-in open() when we are Py3-only.
```

Comment:
Oops, sent https://github.com/web-platform-tests/wpt/pull/28890 and this should be based on it.